backend/handlers/handlers.go

```diff
@@ -961,22 +958,28 @@ func (handlers *Handlers) getAccountSummary(_ *http.Request) (interface{}, error
 		jsonTotals[c.Code()] = handlers.formatAmountAsJSON(coin.NewAmount(total), c, false)
 	}
 
-	truncateLeadingZeroes := func(s []chartEntry) []chartEntry {
-		for i, e := range s {
+	toSortedSlice := func(s map[int64]chartEntry) []chartEntry {
+		var result []chartEntry
+		for _, entry := range s {
+			result = append(result, entry)
```

Comment:
I think this is one of those rare cases where initial slice allocation matters due to a potentially large enough dataset. I would do

```go
var result = make([]chartEntry, len(s))
result[i] = entry
```

Reply:
Done, PTAL.

Reply:
:heavy_check_mark: